mongodb_atlas/metadata.csv

```diff
@@ -30,10 +30,10 @@ mongodb.atlas.system.cpu.mongoprocess.norm.childrenkernel,gauge,,percent,,Normal
 mongodb.atlas.system.cpu.mongoprocess.norm.childrenuser,gauge,,percent,,Normalized CPU children user space for mongodb processes,0,mongodb_atlas,normalized cpu children user space for mongodb processes,
 mongodb.atlas.system.cpu.mongoprocess.norm.kernel,gauge,,percent,,Normalized CPU kernel space for mongodb processes,0,mongodb_atlas,normalized cpu kernel space for mongodb processes,
 mongodb.atlas.system.cpu.mongoprocess.norm.user,gauge,,percent,,Normalized CPU user space for mongodb processes,0,mongodb_atlas,normalized cpu user space for mongodb processes,
-mongodb.atlas.system.cpu.mongoprocess.max.norm.childrenkernel,gauge,,percent,,Max normalized CPU children kernel space for mongodb processes,0,mongodb_atlas,max normalized cpu children kernel space for mongodb processes,
```

Comment:
non-blocking, but this diff breaks my brain. What changed??

Reply:
I wouldn't have touched it, but it's likely that I got out of sync.